lib/axolotl/proto.js

```diff
@@ -80,12 +66,6 @@ Axolotl.prototype.init = function init(other, verify) {
   assert(other.handshake  && other.handshake.PK);
   assert(other.ratchet    && other.ratchet.PK);
 
-  // TODO(kudinkin): Shim this?
-  if (self.state.DHI.getPublic().getX() < other.identity.PK.getX())
-    self.mode = 'alice';
-  else
-    self.mode = 'bob'; 
-
```

Comment:
@yahely a vital decision made over here, actually: we need to assign roles properly prior to starting conversation due to the fact that HS protocol isn't symmetrical.